test/performance/README.md

```diff
@@ -2,23 +2,24 @@
 
 ## Configuring your cluster to run a benchmark
 
-1. Create a namespace or use an existing namespace. Each namespace can be
-   configured with a single benchmark.
+1. Create a namespace `perf-eventing` if not exists. To use a different
+   namespace, please replace all the namespaces in
+   all bash commands and yaml configuration files with your choice.
 
-1. Install Knative eventing by following the steps in
-   https://github.com/knative/eventing/blob/2c6bf0526634804b7ebeee686445901440cc8edd/test/performance/performance-tests.sh#L31
+1. Install Knative eventing and components used in the performance test,
+   such as MT broker, by following the steps in https://github.com/knative/eventing/blob/master/DEVELOPMENT.md.
 
 1. Create a ConfigMap called `config-mako` in your chosen namespace containing
    the Mako config file.
 
 ```
-kubectl create configmap -n <namespace> config-mako --from-file=test/performance/benchmarks/<benchmark>/dev.config
+kubectl create configmap -n perf-eventing config-mako --from-file=test/performance/benchmarks/<benchmark>/dev.config
```

Comment:
for the namespace name, isn't 'eventing'perf' a more suitable name?

Reply:
The namespace is hardcoded in the document and yaml files.